proof/courier.go

```diff
@@ -1187,13 +1187,32 @@ func (c *UniverseRpcCourier) ReceiveProof(ctx context.Context,
 			LeafKey: assetKey,
 		}
 
-		resp, err := c.client.QueryProof(ctx, &universeKey)
+		// Setup proof receive/query routine and start backoff
+		// procedure.
+		receiveFunc := func() ([]byte, error) {
```

Comment:
Ah I see where the blob comes in now. Alternatively, it could just assign to some closure variable? Just thinking about symmetry/uniformity with the interface here.

Reply:
I'll change it to a closure variable. That way the target function signature will always be minimal.